AirCasting/SessionViews/SessionCartViewModel/MeasurementsViewModel.swift

```diff
@@ -0,0 +1,60 @@
+// Created by Lunar on 11/09/2021.
+//
+
+import Foundation
+import CoreLocation
+import CoreData
+
+class ViewModel: NSObject, ObservableObject {
```

Comment:
2nd point here would be that it's really beneficial to have ViewModel protocols. You can see examples already in the app like `StatisticsContainerViewModelable` or `ForgotPasswordViewModel`. It's not much work to do it like that and the benefits are pretty cool. The main one would be that we can stub some static viewmodels instead of "real" ones that connect with the web while automating screenshot taking or implementing UI tests.

Not required tho.